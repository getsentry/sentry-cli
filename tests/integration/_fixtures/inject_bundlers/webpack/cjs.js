"use strict";var __webpack_require__={r:e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})}},__webpack_exports__={};function bar(e,o){e(o)}function foo(e,o){bar(e,o)}function wat(e,o){foo(e,o)}__webpack_require__.r(__webpack_exports__),wat((function(e){throw new Error(e)}),"boop"),module.exports=__webpack_exports__;
//# sourceMappingURL=cjs.js.map