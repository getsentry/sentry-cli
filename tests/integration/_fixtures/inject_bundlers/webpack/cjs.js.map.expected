{"version":3,"file":"webpack.bundle.js","sources":["webpack://sourcemaps-playground/webpack/bootstrap","webpack://sourcemaps-playground/webpack/runtime/make namespace object","webpack://sourcemaps-playground/./src/bar.js","webpack://sourcemaps-playground/./src/foo.js","webpack://sourcemaps-playground/./src/index.js"],"sourceRoot":"","sourcesContent":["// The require scope\nvar __webpack_require__ = {};\n\n","// define __esModule on exports\n__webpack_require__.r = (exports) => {\n\tif(typeof Symbol !== 'undefined' && Symbol.toStringTag) {\n\t\tObject.defineProperty(exports, Symbol.toStringTag, { value: 'Module' });\n\t}\n\tObject.defineProperty(exports, '__esModule', { value: true });\n};","export function bar(fn, msg) {\n  fn(msg);\n}\n","import { bar } from './bar.js';\n\nexport function foo(fn, msg) {\n  bar(fn, msg);\n}\n","import { foo } from './foo.js';\n\nfunction wat(fn, msg) {\n  foo(fn, msg);\n}\n\nwat(function hello(msg) {\n  throw new Error(msg);\n}, 'boop');\n"],"names":["__webpack_require__","exports","Symbol","toStringTag","Object","defineProperty","value","bar","fn","msg","foo","wat","Error"],"mappings":";;AACA,IAAIA,oBAAsB,CCA1BA,EAAyBC,IACH,oBAAXC,QAA0BA,OAAOC,aAC1CC,OAAOC,eAAeJ,EAASC,OAAOC,YAAa,CAAEG,MAAO,WAE7DF,OAAOC,eAAeJ,EAAS,aAAc,CAAEK,OAAO,GAAO,G,uBCLvD,SAASC,IAAIC,EAAIC,GACtBD,EAAGC,EACL,CCAO,SAASC,IAAIF,EAAIC,GACtBF,IAAIC,EAAIC,EACV,CCFA,SAASE,IAAIH,EAAIC,GACfC,IAAIF,EAAIC,EACV,C,2CAEAE,KAAI,SAAeF,GACjB,MAAM,IAAIG,MAAMH,EAClB,GAAG,Q","debug_id":"feff3330-3acf-5df7-9a95-97b3da52cb8a"}